docs/system-admin-guide/manage-work-packages/work-package-status/README.md

```diff
@@ -19,16 +19,17 @@ To learn about the options for configuring the transition from one status to ano
 
 To create a new work package status click the green ***+ New Status*** icon. 
 
-![work package status](image-20200211142716298.png)
-
 A new window will open, where you will be able to specify the following:
 
 1. Name the new work package status.
```

Comment:
As a side note, is it normal that the order of these fields in create status page is different from the order of the columns in the statuses index page?

Reply:
Excellent question. My first response is yes, but I will follow-up to make sure and will update the screenshots if needed.